test/conformance/cts_exe.py

```diff
@@ -2,48 +2,76 @@
 """
  Copyright (C) 2023 Intel Corporation
 
- Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM
+ Exceptions.
```

Comment:
I think the license string shouldn't change.

```suggestion
 Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
```

Reply:
I'm going to roll back the unrelated formatting, but in general: why not? The meaning of the words is still the same